src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -40,6 +41,9 @@ public class BaseRequest : IBaseRequest
 
             this.RequestUrl = this.InitializeUrl(requestUrl);
 
+            this.sdkVersionHeaderName = CoreConstants.Headers.SdkVersionHeaderName;
```

Comment:
Yes, they should. The specific use case right now is we're going to consume this core project in the OneDrive SDK to reduce duplicate code and those telemetry headers are completely different. That said, we want to encourage the newer format for other SDKs. That's why these are standard variables while prefix is a property.